frontend/src/data/tasks.json

```diff
@@ -1259,7 +1241,7 @@
             "linkEn": "https://www.canada.ca/en/services/benefits/publicpensions/cpp/cpp-international/while-receiving.html",
             "linkFr": "https://www.canada.ca/fr/services/prestations/pensionspubliques/rpc/rpc-internationales/pendant-que-vous-recevez.html",
             "textEn": "Lived or living outside Canada - Pensions and benefits: While on pension and benefits",
-            "textFr": "Personnes ayant vécu ou vivant à l'étranger - Pensions et prestations Pendant que vous recevez pensions et prestations"
+            "textFr": "Personnes ayant vécu ou vivant à l'étranger – Pensions et prestations Pendant que vous recevez pensions et prestations"
```

Comment:
Change this hypen type `–` with a regular hypen `-`.

Reply:
Changed `–` to `-`